.vscode/tasks.json

```diff
@@ -9,5 +9,10 @@
         "reveal": "never",
       },
     },
+    {
+      "type": "npm",
+      "label": "build",
+      "script": "build",
+    },
```

Comment:
Running the `Launch Setting` task resulted in an error. Adding this task fixed the issue